molecule/rolling_upgrade/hosts.yml

```
@@ -0,0 +1,115 @@
+---
+cluster:
+  vars:
+    # common connection opts
+    ansible_user: root
+    ansible_connection: docker
+    become: true
+    become_user: root
+
+    # common cartridge opts
+    cartridge_app_name: myapp
+    cartridge_cluster_cookie: secret-cookie
+
+    cartridge_multiversion: true
+
+    cartridge_enable_tarantool_repo: false
+    cartridge_configure_systemd_unit_files: false
+    cartridge_configure_tmpfiles: false
+    cartridge_install_tarantool_for_tgz: false
+
+    cartridge_bootstrap_vshard: true
+
+    cartridge_failover_params:
+      mode: stateful
+      state_provider: stateboard
+      stateboard_params:
+        uri: vm1:4001
+        password: secret-stateboard
+
+    cartridge_custom_scenarios:
+      update_and_restart_instance:
+        - update_instance
+        - restart_instance
+        - wait_instance_started
+
+  # instances
+  hosts:
+    core-1:
+      config:
+        advertise_uri: 'vm1:3301'
+        http_port: 8151
```

Let's use the ports similarly to other tests:
Tarantool port = `3<machine-id>0<instance-id>`
HTTP port = `8<machine-id>0<instance-id>`

So, this place will have the following ports:
```suggestion
        advertise_uri: 'vm1:3101'
        http_port: 8101
```